src/socket-types.ts

```diff
@@ -206,6 +206,35 @@ export enum ChainType {
   default = "default",
 }
 
+export enum FinalityBucket {
+  fast = 1,
+  medium = 2,
+  slow = 3,
+}
+
+export enum FinalityType {
+  bucket = 1,
+  block = 2,
+  time = 3,
+}
```

Comment:
FinalityType enum is used in 2 places and prone to confusion.
First is to describe type in transmission params
Second is to describe type of bucket in ChainFinalityInfo